example/src/screens/Room.tsx

```diff
@@ -1,181 +1,187 @@
+      e.preventDefault();
+      // TODO(@skyman503): Navigate to `LeaveScreen` when it is implemented.
+      // Check whether beforeRemove event was triggered by disconenct button.
+      if (e.data.action.source) {
+        navigation.dispatch(e.data.action);
+      }
+    };
 
-  const [areSettingsOpen, setAreSettingsOpen] = useState<boolean>(false);
+    navigation.addListener('beforeRemove', handleBeforeRemoveEvent);
 
-  const isFocusedParticipantSpeaking =
-    focusedParticipant?.tracks.find((t) => t.type === 'Audio')?.vadStatus ===
-    Membrane.VadStatus.Speech;
+    return () =>
+      navigation.removeListener('beforeRemove', handleBeforeRemoveEvent);
+  }, [navigation]);
 
   const disconnect = useCallback(() => {
     mbDisconnect();
     navigation.goBack();
   }, []);
 
+  const switchCamera = useCallback(() => {
+    Membrane.flipCamera();
+  }, []);
+
+  const getWidthWhenManyParticipants = () => {
+    return Math.min(videoViewWidth, smallScreenVideoWidth);
+  };
+
+  const getStylesForParticipants = (participants: Membrane.Participant[]) => {
+    return [
+      styles.participant,
+      participants.length > FLEX_BRAKPOINT
+        ? {
+            width: getWidthWhenManyParticipants(),
+          }
+        : {
+            flex: 1,
+            maxHeight: width - 2 * PADDING_BETWEEN_PARTICIPANTS,
+            maxWidth: width - 2 * PADDING_BETWEEN_PARTICIPANTS,
+          },
+    ];
+  };
+
   return (
-    <View style={styles.flex}>
-      <View style={styles.flex}>
-        {!!focusedTrack && !!focusedParticipant && (
-          <View
-            style={[
-              styles.focusedParticipantContainer,
-              isFocusedParticipantSpeaking ? styles.vadSpeech : {},
-            ]}
-          >
-            <Membrane.VideoRendererView
-              trackId={focusedTrack.id}
-              style={styles.focusedParticipant}
-              videoLayout={Membrane.VideoLayout.FIT}
-            />
-            <Text style={styles.displayName}>
-              {focusedParticipant.metadata.displayName}
-            </Text>
-            {!!areSettingsOpen && (
-              <View style={styles.settingsWrapper}>
-                <Settings
-                  participant={focusedParticipant}
-                  track={focusedTrack}
-                />
-              </View>
-            )}
-            <View style={styles.disabledIconsContainer}>
-              {!focusedParticipant.tracks.find((t) => t.type === 'Audio')
-                ?.metadata.active && <Icon name="Microphone-off" size={24} />}
-              {!focusedTrack.metadata.active && (
-                <Icon name="Cam-disabled" size={24} />
+    <BackgroundAnimation>
+      <View style={styles.container}>
+        <View style={styles.header}>
+          <View style={styles.headerTitle}>
+            <Typo variant="h4">{roomName}</Typo>
+          </View>
+          <View style={styles.headerIcon}>
+            <Pressable onPress={switchCamera}>
```

Comment:
maybe use touchable opacity to provide some feedback to the user?

Reply:
Done.